File: src/main/java/top/zbeboy/isy/web/vo/data/college/CollegeVo.java
package top.zbeboy.isy.web.vo.data.college;

import lombok.Data;

import javax.validation.constraints.Min;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by lenovo on 2016/9/22.
 */
@Data
public class CollegeVo {
    private Integer collegeId;
    @NotNull
    @Size(max = 200)
    private String collegeName;
    private Byte collegeIsDel;
    @NotNull
    @Min(1)
    private Integer schoolId;
    @NotNull
    @Size(max = 500)
    private String  collegeAddress;
    @NotNull
    @Size(max = 20)
    private String  collegeCode;
}

File: src/main/java/top/zbeboy/isy/web/vo/data/college/CollegeVo.kt
package top.zbeboy.isy.web.vo.data.college

import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2017-12-01 .
 **/
open class CollegeVo {
    var collegeId: Int? = null
    @NotNull
    @Size(max = 200)
    var collegeName: String? = null
    var collegeIsDel: Byte? = null
    @NotNull
    @Min(1)
    var schoolId: Int? = null
    @NotNull
    @Size(max = 500)
    var collegeAddress: String? = null
    @NotNull
    @Size(max = 20)
    var collegeCode: String? = null
}